tests/actions/test_committee.py

```diff
@@ -28,7 +28,8 @@ class CommitteeCreateActionUnitTester(BaseCommitteeCreateActionTester):
     def setUp(self) -> None:
         super().setUp()
         self.action = CommitteeCreate(PermissionTestAdapter(), DatabaseTestAdapter())
-        self.action.user_id = 7668157706  # This user has perm COMMITTEE_CAN_MANAGE.
+        # This user has perm COMMITTEE_CAN_MANAGE.
```

Comment:
Please revert this change. It is much easier for me to copy past a user_id with its short description in test code, if it remains in one line. Please revert also in all other cases.